src/AltinnCore/Common/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -91,47 +70,27 @@ public async Task<AltinnCore.RepositoryClient.Model.User> GetCurrentUser(string
         /// <summary>
         /// Create repository
         /// </summary>
-        /// <param name="giteaSession">the gitea session</param>
         /// <param name="org">the organisation</param>
         /// <param name="createRepoOption">the options for creating repository</param>
         /// <returns>The newly created repository</returns>
-        public async Task<Repository> CreateRepositoryForOrg(string giteaSession, string org, CreateRepoOption createRepoOption)
+        public async Task<Repository> CreateRepositoryForOrg(string org, CreateRepoOption createRepoOption)
         {
             AltinnCore.RepositoryClient.Model.Repository repository = null;
             DataContractJsonSerializer serializer = new DataContractJsonSerializer(typeof(AltinnCore.RepositoryClient.Model.Repository));
 
-            Uri giteaUrl = new Uri(_settings.ApiEndPoint + "/org/" + org + "/repos");
-            Cookie cookie = new Cookie(_settings.GiteaCookieName, giteaSession, "/", _settings.ApiEndPointHost);
-
-            if (Environment.GetEnvironmentVariable("GiteaApiEndpoint") != null && Environment.GetEnvironmentVariable("GiteaEndpoint") != null)
-            {
-                giteaUrl = new Uri(Environment.GetEnvironmentVariable("GiteaApiEndpoint") + "/org/" + org + "/repos");
-                cookie = new Cookie(_settings.GiteaCookieName, giteaSession, "/", Environment.GetEnvironmentVariable("GiteaEndpoint"));
-            }
-
-            CookieContainer cookieContainer = new CookieContainer();
-            cookieContainer.Add(cookie);
-            HttpClientHandler handler = new HttpClientHandler() { CookieContainer = cookieContainer };
-
-            using (HttpClient client = new HttpClient(handler))
+            Uri endpointUrl = new Uri(_settings.ApiEndPoint + "/org/" + org + "/repos");
```

Comment:
Can we not use GetApiBaseUrl() here as well?

Reply:
Already fixed